src/pybind/mgr/dashboard/frontend/src/app/shared/components/bottom-links/bottom-links.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'cd-bottom-links',
+  templateUrl: './bottom-links.component.html',
+  styleUrls: ['./bottom-links.component.scss']
+})
+export class BottomLinksComponent {
+  docTitles: any[] = [
```

Comment:
Think the following is more apropriate:
```suggestion
  docLinks: any[] = [
```
or
```suggestion
  docItems: any[] = [
```

Reply:
Done.